azure_mgmt_graph/lib/generated/azure_mgmt_graph/applications.rb

```diff
@@ -131,15 +131,38 @@ def create_async(parameters, custom_headers = nil)
     # Lists applications by filter parameters. Reference:
     # http://msdn.microsoft.com/en-us/library/azure/hh974476.aspx
     #
+    # @param api_version [String] Client Api Version.
```

Comment:
This line https://github.com/Azure/autorest/blob/master/src/generator/AutoRest.Ruby.Azure/Templates/AzureMethodTemplate.cshtml#L25 should be https://github.com/Azure/autorest/blob/master/src/generator/AutoRest.Ruby/Templates/MethodTemplate.cshtml#L18 to ignore constants from the method comments.